helpers/add-many.js

```diff
@@ -1,11 +1,11 @@
 'use strict'
 
 var toDoc = require('../utils/to-doc')
+var addTimestamps = require('../utils/add-timestamps')
 
 module.exports = function addMany (objects) {
-  var docs = objects.map(toDoc)
```

Comment:
Think this original docs set me on that path, stopped me from seeing the simpler solution.